ansible-scylla-node/molecule/rockylinux9/molecule.yml

```diff
@@ -0,0 +1,100 @@
+---
+dependency:
+  name: galaxy
+  enabled: false
+  options:
+    role-file: requirements.yml
+driver:
+  name: docker
+platforms:
+  - name: node1
+    image: geerlingguy/docker-rockylinux9-ansible
+    privileged: true
+    command: /usr/lib/systemd/systemd
+    cgroupns_mode: host
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+    groups:
+      - scylla
+    docker_networks:
+      - name: scylla
+        ipam_config:
+          - subnet: "10.11.0.0/16"
+            gateway: "10.11.0.254"
+    networks:
+      - name: "scylla"
+        ipv4_address: "10.11.0.2"
+  - name: node2
+    image: geerlingguy/docker-rockylinux9-ansible
+    privileged: true
+    command: /usr/lib/systemd/systemd
+    cgroupns_mode: host
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+    groups:
+      - scylla
+    networks:
+      - name: "scylla"
+        ipv4_address: "10.11.0.3"
+  - name: node3
+    image: geerlingguy/docker-rockylinux9-ansible
```

Comment:
Please, add an explanation about the motivation for not using a vanilla Rocky image as an inline comment.

Reply:
Done